src/Fieldtypes/Bard.php

```diff
@@ -113,6 +113,7 @@ protected function configFieldItems(): array
                         'instructions' => __('statamic::fieldtypes.bard.config.enable_input_rules'),
                         'type' => 'toggle',
                         'default' => true,
+                        'validate' => 'accepted_if:smart_typography,true',
```

Comment:
Can you remove this change from this PR? It's covered by your other PR, #9555 😄

Reply:
Ugh, I suck at git. Lets see if I can figure this out.

Reply:
Done